src/widgets/backuplistwidgetitem.cpp

```diff
@@ -91,7 +99,7 @@ void BackupListWidgetItem::updateDirDetail(quint64 size, quint64 count)
 {
     _size  = size;
     _count = count;
-    _ui.detailLabel->setText(QString::number(_count) + tr(" items totalling ")
+    _ui.detailLabel->setText(QString::number(_count) + tr(" items, ")
```

Comment:
No, I would assume that "files" does not include directories.  As for "items"... that is the term used by the windows file browser, the OSX finder, the Ubuntu file browser... I think it's a fine term for this!